### PR TITLE
Adds LCP guidance.

### PR DESCRIPTION
**Pull request by malchata**
Changes proposed in this pull request:

- Adds guidance around LCP on the [serve responsive images guide](https://web.dev/serve-responsive-images/).

### CODE REVIEW

File: src/site/content/en/fast/serve-responsive-images/index.md
Comment (tunetheweb) [inline src/site/content/en/fast/serve-responsive-images/index.md:34]: Not sure about this first sentence. "It's more than just faster loading images. For example you can have a faster loading LCP image". That's just a faster loading image isn't it?

I do agree with the phrase, but think we should give a better example (while also leaving the important LCP example in place).

What about saying it can affect the load time of the rest of the page as large images can hold up all other resources due to bandwidth constraints? And it also impacts Core Web Vitals. For example if your LCP...

File: src/site/content/en/fast/serve-responsive-images/index.md
Comment (tunetheweb) [inline src/site/content/en/fast/serve-responsive-images/index.md:35]:
```suggestion
example, if your page's [LCP element](/lcp/#what-elements-are-considered) is an image,
```

File: src/site/content/en/fast/serve-responsive-images/index.md
Comment (malchata) [reply src/site/content/en/fast/serve-responsive-images/index.md:34]: The point I guess I was (poorly) trying to make is that the effect extends to the LCP metric, which is a measure of perceptual performance. I'll have to workshop this.

Comment (malchata) [reply src/site/content/en/fast/serve-responsive-images/index.md]: I revised the language in this paragraph, @tunetheweb.

**Review by tunetheweb — APPROVED** [review]

File: src/site/content/en/fast/serve-responsive-images/index.md
Comment (philipwalton) [inline src/site/content/en/fast/serve-responsive-images/index.md:38]: This phrase ("Lower resource load times will lower the load time for an LCP image") feels redundant/repetitive to me. Also, it's not guarantee that a lower resource load time will improve LCP.

I'd rephrase this: "A lower LCP resource load time will generally result in a better overall LCP score."

**Review by philipwalton — APPROVED** [review]
LGTM (modulo one comment).

**Pull request merged**
